marklogic-data-hub-central/ui/src/components/common/hc-tag/hc-tag.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./hc-tag.module.scss";
+import iconCloseTag from "../../../assets/icon_closeTag.svg";
+
+interface Props {
+  label: string;
+  ariaLabel?: string;
+  style?: React.CSSProperties;
+  className?: string;
+  dashed?: boolean;
+  color?: string;
+  closable?: boolean;
+  visible?: boolean;
+  onClose?: () => void;
+}
+
+const setTagColor = (color: string) => {
+
+  let colorAux = color.toLowerCase();
+  let tagColor = "";
+
+  switch (colorAux) {
```

Comment:
I think this could be replaced with a map, with the structure:
Key=colorAux
Value=style.spanTag...

And this switch will replaced with just return map[colorAux]

Reply:
I agree Federico, it's more clean. 
And now i added an improvement to the color syntax:   color?: "black" | "blue" | "green" | "grey" | "magenta" | "red" | "yellow";